src/content/docs/guides/JSON/0-json_intro.mdx

```diff
@@ -192,6 +221,26 @@ else
   FreeJson(gameData);
 ```
 
+</TabItem>
+<TabItem label="Python">
+
+  ```python
+from splashkit import *
+
+def main():
+    game_data = json_from_file("game_data.json")
+    
+    if json_has_key(game_data, "gameTitle"):
+        game_title = json_read_string(game_data, "gameTitle")
+        print("Game Title:", game_title)
+    else:
+        print('Key "gameTitle" not found.')
```

Comment:
This needs to use SplashKit's "write_line" function to follow the same format as the other code.